dashboard/dashapi/dashapi.go

```diff
@@ -623,3 +627,49 @@ func (dash *Dashboard) queryImpl(method string, req, reply interface{}) error {
 	}
 	return nil
 }
+
+type RecipientType int
+
+const (
+	To RecipientType = iota
+	Cc
+)
+
+func (t RecipientType) String() string {
+	return [...]string{"To", "Cc"}[t]
+}
+
+type RecipientInfo struct {
+	Address mail.Address
+	Type    RecipientType
+}
+
+type Recipients []RecipientInfo
+
+func (r Recipients) GetEmails(filter string) []string {
```

Comment:
This does not seem to be used. I can imagine we will need this in the dashboard app maybe? If so, it better go into dashboard/app, nobody else will use it. This package is more like "interface declarations", we try to keep it free of large chunks of code.